README.md

```diff
@@ -132,6 +132,11 @@ export LD_LIBRARY_PATH=/<YOUR_PRODUCER_SDK_CPP_DOWNLOAD>/amazon-kinesis-video-st
 This should resolve native library loading issues.
 
 ## Release Notes
+
+### Release 2.0.0 (27 May 2020)
```

Comment:
Is there a good reason to bump up the major version? Perhaps 1.10.0?

Reply:
The reason for the major revision bump up is because the producer SDK is bumped up. Bumping up to establish that our base SDK has gone through a major rev. But, yes. Other than the docker scripts breaking between revs, I do not see anything else that might break. Let me change it to 1.10.0.

Reply:
I am somewhat split and gravitate more towards not changing the major version. The minor version update is for the C producer and CPP producer as they do take advantage of the functionality that we added in lower layers and some are not fully compatible. Whereas Java SDK is based on PIC and should be fully compatible.